doc/man7/EVP_MAC-Poly1305.pod

```diff
@@ -49,6 +49,11 @@ Gets the MAC size.
 The "size" parameter can also be retrieved with with EVP_MAC_CTX_get_mac_size().
 The length of the "size" parameter should not exceed that of an B<unsigned int>.
 
+=head1 NOTES
+
+The OpenSSL implementation of the Poly 1305 MAC corresponds to RFC 7539.
+As noted in RFC 8439, the key should not be reused.
```

Comment:
This is quite confusing. There is 7539 and 8439. If I'm reading this correctly we conform to the earlier one, but we're repeating advice from the later one?

Reply:
The security advice is relevant to both.  I'll see if I can reword things more clearly.

Reply:
Updated.